devstart

```diff
@@ -11,7 +11,7 @@ echo '        \/          \/     \/     \/                            \/
 echo ''
 
 echo "[MumukiDevstart::ClassroomApi] Installing dependencies...."
-bundle install --quiet
+bundle install
```

Comment:
I don't think removing this flag is a good. Most of the times this will display the installed gems, even if you have not done any changes, so except after a fresh install it will provide no real information. 

If you want to improve this script, I'd suggest changing it so that it only displays that information only when it will effectively install something.